pkg/cmd/cli/cmd/env.go

```diff
@@ -324,6 +332,11 @@ func RunEnv(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra.Comman
 
 	failed := false
 	for i, info := range infos {
+		shortOutput := cmdutil.GetFlagString(cmd, "output") == "name"
```

Comment:
move `shortOutput := cmdutil.GetFlagString(cmd, "output") == "name"` outside the loop